src/deye_processor_factory.py

```diff
@@ -28,21 +28,21 @@
 from deye_sensor import Sensor
 from deye_plugin_loader import DeyePluginContext, DeyePluginLoader
 from deye_multi_inverter_data_aggregator import DeyeMultiInverterDataAggregator
-
+from deye_modbus import DeyeModbus
```

Comment:
@belowm Remove this extra import, otherwise the PR check will not pass and will block the merge of the PR